### PR TITLE
RatingDisplay value token update

### LINKED ISSUE #31033: Update spacing tokens for v9 Rating rating value
This issue is to track a design update to the `Rating` value spacing tokens for each `size`.

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR updates the value of the `marginLeft` tokens for the `RatingDisplay` value slot with the new figma values. Fixes #31033

### CODE REVIEW

File: packages/react-components/react-rating/src/components/RatingDisplay/useRatingDisplayStyles.styles.ts [hunk 1]
```diff
@@ -21,18 +21,20 @@ const useRootClassName = makeResetStyles({
 
 const useLabelClassName = makeResetStyles({
   color: tokens.colorNeutralForeground1,
-  marginLeft: tokens.spacingHorizontalXXS,
+  marginLeft: tokens.spacingHorizontalXS,
```
Comment (mltejera) [inline packages/react-components/react-rating/src/components/RatingDisplay/useRatingDisplayStyles.styles.ts:24]: Someone pointed out logical properties on one of my PR's a little bit ago.... slightly hard to reason about, but makes handling RTL issues easier.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount_base.html">316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount.html">295</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">37</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Avatar-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Avatar-mount.html">650</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount_base.html">316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount.html">295</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Button-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Field-mount_base.html">1109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Field-mount.html">1117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProvider-mount_base.html">721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProvider-mount.html">721</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-mount_base.html">83</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-mount.html">93</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">37</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">76</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/MakeStyles-mount_base.html">858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/MakeStyles-mount.html">854</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Persona-mount_base.html">1777</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/Persona-mount.html">1714</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/SpinButton-mount_base.html">1392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31034/perf-test-react-components/SpinButton-mount.html">1365</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-rating-deec31e1-3da3-442d-a5f3-dc7a74057fd0.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-rating-deec31e1-3da3-442d-a5f3-dc7a74057fd0.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31034&commitId=dd2d6ec837c9b6c10d0f6e842a7d9c14a9b21bbb&env=prod&clientType=FLUENTUI)


  <details><summary><b>RatingDisplay Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Compact.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|59| Changed|
|RatingDisplay Converged.custom circle icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|166| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|124| Changed|
|RatingDisplay Converged.custom square icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|70| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|261| Changed|
|RatingDisplay Converged.Marigold with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|124| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-rating/src/components/RatingDisplay/useRatingDisplayStyles.styles.ts (hunk 1 above)
Comment (tomi-msft) [reply packages/react-components/react-rating/src/components/RatingDisplay/useRatingDisplayStyles.styles.ts:24]: This is a good callout. I will make a separate PR to update the general logic properties.

**Pull request merged**